pandas/tests/window/test_ewm.py

```diff
@@ -231,3 +230,14 @@ def test_float_dtype_ewma(func, expected, float_numpy_dtype):
     result = getattr(e, func)()
 
     tm.assert_frame_equal(result, expected)
+
+
+def test_times_string_col_deprecated():
+    # GH 43265
+    data = np.arange(10.0)
+    data[::2] = np.nan
+    df = DataFrame({"A": data, "time_col": date_range("2000", freq="D", periods=10)})
+    with tm.assert_produces_warning(FutureWarning, match="Specifying times"):
```

Comment:
do we have any tests with times as an array (you do above in test_ewma_with_times_equal_spacing but you are also hitting the nuiscance path

Reply:
This test passes `times` as an array


https://github.com/pandas-dev/pandas/blob/a6943ae49f79452da6ca5f1c3e008ded4e0e2c85/pandas/tests/window/test_ewm.py#L126-L136